--- web/original/opt_phpbrew_php_php-7_etc_php.ini
+++ web/patched/opt_phpbrew_php_php-7_etc_php.ini
@@ -618,7 +618,7 @@
 ; Development Value: "GPCS"
 ; Production Value: "GPCS";
 ; http://php.net/variables-order
-variables_order = "GPCS"
+variables_order = "EGPCS"
 
 ; This directive determines which super global data (G,P & C) should be
 ; registered into the super global array REQUEST. If so, it also determines
@@ -669,7 +669,7 @@
 ; Its value may be 0 to disable the limit. It is ignored if POST data reading
 ; is disabled through enable_post_data_reading.
 ; http://php.net/post-max-size
-post_max_size = 8M
+post_max_size = 64M
 
 ; Automatically add files before PHP document.
 ; http://php.net/auto-prepend-file
@@ -822,7 +822,7 @@
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 64M
 
 ; Maximum number of files that can be uploaded via a single request
 max_file_uploads = 20
@@ -936,7 +936,7 @@
 [Date]
 ; Defines the default timezone used by the date functions
 ; http://php.net/date.timezone
-;date.timezone =
+date.timezone = "Europe/Berlin"
 
 ; http://php.net/date.default-latitude
 ;date.default_latitude = 31.7667
